app/src/docs/v1.api-spec.yaml

```diff
@@ -17,30 +17,348 @@ security:
+      operationId: GetObjectPermissions
+      tags:
+        - Permissions
+      parameters:
+        - $ref: '#/components/parameters/PathMessageId'
+      responses:
+        '200':
+          description: Returns the object information
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ObjectDetails'
+        '401':
+          $ref: '#/components/responses/UnauthorizedError'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '422':
+          $ref: '#/components/responses/UnprocessableEntity'
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+
+  '/{objId}/permissions/{userId}':
+    post:
+      summary: Grants object permissions to a specific user
+      description: >-
+        This endpoint will update the permissions of a specific user
+      operationId: PostUpdateUserPermissions
+      tags:
+        - Permissions
+      parameters:
+        - $ref: '#/components/parameters/PathMessageId'
+        - $ref: '#/components/parameters/PathUserId'
+      responses:
+        '200':
+          description: Returns the object information
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ObjectDetails'
+        '401':
+          $ref: '#/components/responses/UnauthorizedError'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '409':
+          $ref: '#/components/responses/Conflict'
+        '422':
+          $ref: '#/components/responses/UnprocessableEntity'
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ValidationError'
+    delete:
+      summary: Deletes object permissions for a specific user
+      description: >-
+        This endpoint will delete the permission of an Object with a specific to the objId
+      operationId: DeleteObjectPermissions
+      tags:
+        - Permissions
+      parameters:
+        - $ref: '#/components/parameters/PathMessageId'
+        - $ref: '#/components/parameters/PathUserId'
+      responses:
+        '201':
+          description: Returns udpate Object response.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MergeResponse'
+        '401':
+          $ref: '#/components/responses/UnauthorizedError'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '409':
+          $ref: '#/components/responses/Conflict'
+        '422':
```

Comment:
maybe check the chefs api spec that contains similar operations.
i dont think we would have a 422 or 409 in this api.
at this point anyway

Reply:
removing both 409 and 422 responses